### PR TITLE
chore: assign individual CODEOWNERS

### PR DESCRIPTION
**Pull request by makkes**
This will make it much easier to get chart changes merged.

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -7,9 +7,23 @@
...
+/stable/dex-controller @mesosphere/sig-ksphere-security
+/stable/dispatch @mesosphere/devx
+/stable/gcpdisk-csi-driver @mesosphere/sig-ksphere-cluster
+/stable/gcpdiskprovisioner @mesosphere/sig-ksphere-cluster
+/stable/kommander @mesosphere/sig-ksphere-ui @mesosphere/sig-ksphere-multi-cluster
+/stable/kommander-karma @mesosphere/sig-ksphere-observability
+/stable/kommander-thanos @mesosphere/sig-ksphere-observability
+/stable/localvolumeprovisioner @mesosphere/sig-ksphere-cluster
+/stable/mtls-proxy @mesosphere/sig-ksphere-observability
+/stable/opsportal @mesosphere/sig-ksphere-ui
```
Comment (juliangieseke) [inline CODEOWNERS]:
```suggestion
/stable/opsportal @mesosphere/sig-ksphere-ui @mesosphere/sig-ksphere-security 
```

Comment (juliangieseke) [reply CODEOWNERS]: we're just shipping our stuff as subchart, not owning the addon

Comment (juliangieseke) [reply CODEOWNERS]: security is a guess, no idea who "owns" that thing?!

**Comment by shaneutt** [comment]
Let's double check with @jaxesn before merging? but otherwise LGTM.

**Comment by jaxesn** [comment]
As mentioned in slack, I fine with this, using groups or not, its up to you guys.

**Pull request merged**